### DISCUSSION TITLE
[Question] Building extension for wikilinking

### DISCUSSION THREAD

**Discussion by eenpadvinder** [standardnotes/forum#2615] [General] [answered]
Hi! I was wondering if it would be possible to build my own extension (probably extended from basic markdown editor) to allow for following `[[wikilinks]]`. Before I dive into that, I just wanted to check if it would be technically possible/allowed for an extension to open other notes or start a search.

**Comment by myreli** [comment, marked as answer]
Due to the secure plugin architecture, it is not possible for 3rd party extensions to access other notes. Instead, they are sandboxed to the specific content to which they were authorized.

However, since the [launch of the built-in super editor](https://www.youtube.com/watch?v=bkp6nrGqjUM&t=370s) bi-directional linking is now supported through the `@` shortcut. You can link notes, tags and files trough the same shortcut. 

(I'm sorry this reply is coming super late, we are triaging older issues and some questions were lost)
